package.json

```diff
@@ -107,11 +107,11 @@
     "@eslint-community/eslint-plugin-eslint-comments": "^4.3.0",
     "@types/estree": "^1.0.5",
     "@types/node": "^20.11.5",
-    "@wdio/browser-runner": "8.36.0",
-    "@wdio/cli": "^8.14.6",
-    "@wdio/concise-reporter": "^8.14.0",
-    "@wdio/globals": "^8.14.6",
-    "@wdio/mocha-framework": "^8.14.0",
+    "@wdio/browser-runner": "^8.38.3",
+    "@wdio/cli": "^8.38.2",
+    "@wdio/concise-reporter": "^8.38.2",
+    "@wdio/globals": "^8.38.2",
+    "@wdio/mocha-framework": "^8.38.2",
```

Comment:
Thanks for the update, but the browser tests are still failing. [Reference](https://github.com/eslint/eslint/actions/runs/9377636509/job/25819524336?pr=18558)